parteh/PRTGenericMod.F90

```diff
@@ -1124,6 +1133,39 @@ function GetBurned(this, organ_id, element_id, position_id) result(burned_val)
       return
    end function GetBurned
 
+    ! =========================================================================
+    
+    function GetHerbivory(this, organ_id, element_id, position_id) result(herbivory_val)
+
+      ! This function is very very similar to GetBurned
```

Comment:
In fact it looks identical, aside from `herbivory` replacing `burned`. Could these be replaced with a single function? Named, e.g., `GetDisturbed`.